mpyg321/mpyg321.py

```diff
@@ -1,6 +1,28 @@
 import pexpect
 from threading import Thread
 
+def check_installed():
+    """Checks if mpg321 or mpg123 is installed"""
+    import os
+
+    path = os.environ["PATH"].split(os.pathsep)
+    for directory in path:
+        mpg321_path = os.path.join(directory, "mpg321")
+        mpg123_path = os.path.join(directory, "mpg123")
+
+        if os.path.exists(mpg321_path):
+            return "mpg321"
+        elif os.path.exists(mpg123_path):
+            return "mpg123"
+
+    return False
+
+if not(check_installed()):
```

Comment:
This should be done inside the __init__ function. If you need the player_command variable outside the init function (which I am not sure you do), then save it in a member of the class. Global variables are really not a good practice in general :)

Reply:
I've moved it to within the __init__ method. I'm also going to change it slightly as per your suggestion.